### DISCUSSION TITLE
More incentives to transport passengers

### DISCUSSION THREAD

**Discussion by victorleblais** [voxeltycoon/issues#353] [General]
_Suggestion inspired by PinStratsDan#2259 on Discord_

> At the moment I don’t feel compelled to make effort in transporting passengers. It feels like an optional add on because there is no prompting from the game to service an area i.e. businesses go bankrupt if you don’t supply them.  My suggestion is that your ratings in a town should go down if you don’t service the passengers, which should then negatively affect how much you make from delivering goods.

> There is also no research goal linked to passengers as far as I can tell. Maybe add something like "transfer a X passengers" or "provide passenger service in X cities".

This feeling is also expressed in this review of the Early Access launch version by pixel. : https://youtu.be/37qQv6_N1og?t=224

If you have a similar feeling and if you have ideas, please share them !

**Comment by noahjsc** [comment]
Transporting passengers has a lot of incentive in my personal opinion. Town growth is influenced by passenger transport. Especially tourist type towns. Town growth is important as it influences demand for products as such having larger towns allows you to move greater amount of resources. Players who desire to min/max or just grow towns are definitely rewarded for transporting passengers. Furthermore, passenger transport arguable has the greatest possible income possible for a route(not per vehicle). However to reach that you need a high population city.

**Comment by mathwizi2005** [comment]
I find if you just need the town to grow, just invest in buildings instead. After you unlock a number of regions, supplying routes for every possible destination gets tedious and, due to the sheer infrastructure needed, slowly gets less profitable due to traffic problems supplying terminals on denser cities.

That being said, going the route of punishing the player for not providing passenger transport would drag out the early and mid game longer, if not bankrupt some players if a strict hierarchy of construction isn't followed.

This could be fixed by reducing the sheer transport demand needed / having a destination request list with how many passengers want to do to X town / supporting **intra**city transport to offset the demand

Just an FYI, as a new player, I still do not know how to setup terminal systems and supply routes for those terminals and with no tutorials ingame that show its a possibility, you will find players just plopping bus stops around a city, expecting passenger generation and get confused why their bus routes are 1/10th the profits of cargo delivery (current observation and from a player that is use to the 'Deliver cargo from Station A to Station B' method)

**Comment by asiaclonk** [comment]
If there is going to be a passenger transport specific reputation for towns, you could reward sufficient passenger transport for a certain duration with town specific favors that usually requires population approval.
For example:
* Relocating one significant building (business, landmarks, etc.) to a nearby spot (limited by a radius) of your choice for a price
* A discount on demolishing town structures
* Sponsoring a new business of your choice within range of the town hall if there is a slot left.

You should only be able to accumulate and hold onto one favor per town at a time. Significant favor could be rewarded for maintaining excellent transport rating during events such as festivals taking place at parks, major sports at stadiums, etc.

**Comment by jackfirth** [comment]
I also don't find passenger transit much fun to set up currently. I don't think there are enough clear incentives to do so, as increasing town growth is the only benefit, but it's a subtle indirect benefit that's not explained to the player. Additionally, the passenger transport gameplay UX is too different from the industrial goods gameplay UX. So with it being pretty different to set up and with there not being a clear motivation to do so, I just end up ignoring passenger transit entirely.

What if towns had businesses that demand tourists from other towns instead of goods? For instance, a town could grow a public beach, parks, museums, or other tourism-focused structures. They could work the same way businesses demanding goods do except you've got to bring passengers interested in those activities to them. Getting rid of the distinction between passenger and freight stations may also help since I think that just discourages players further from trying to set up both.
